web/modules/custom/dpl_event/src/Services/EventRestMapper.php

```diff
@@ -252,6 +253,22 @@ private function getDateField(string $field_name): ?DateTime {
     return $date;
   }
 
+  /**
+   * Get multiple string values as an array output.
+   *
+   * @return string[]|null
+   *   An array of string values.
+   */
+  private function getMultiValue(string $field_name): ?array {
+    $strings = $this->getValue($field_name);
+
+    if (empty($strings)) {
+      return NULL;
+    }
+
+    return explode(', ', $strings);
```

Comment:
Please consider if this is an appropriate way to handle multivalue fields. 

As I read it, if one of the value contain `, ` then it will be treated as a multivalue. That does not seem right. Is there a better way to go about this?

If we go with this approach I would appreciate a reference to why `, ` is an appropriate separator.

`event_partners` is a multivalue field and to me it is not obvious that multiple values are separated by `, ` by Drupal Core/`$field->getString()`.

Reply:
It was an attempt to reuse the custom getValue() method, but, i didnt consider that strings might actually contain `, `. 
I've updated it, to use the Drupal `getValue` method instead, and using `array_column` to turn it into a simple array.